dvc/dvcfile.py

```diff
@@ -29,6 +30,29 @@
 TAG_REGEX = r"^(?P<path>.*)@(?P<tag>[^\\/@:]*)$"
 
 
+class MultiStageFileLoadError(DvcException):
+    def __init__(self):
+        super().__init__("Cannot load multi-stage file.")
+
+
+def _serialize_stage(stage):
+    return {
+        stage.name: {
+            key: value
+            for key, value in {
+                stage.PARAM_CMD: stage.cmd,
+                stage.PARAM_WDIR: stage.resolve_wdir(),
+                stage.PARAM_DEPS: [d.def_path for d in stage.deps],
+                stage.PARAM_OUTS: [o.def_path for o in stage.outs],
+                stage.PARAM_LOCKED: stage.locked,
+                stage.PARAM_ALWAYS_CHANGED: stage.always_changed,
+                "persist": [o.def_path for o in stage.outs if o.persist],
+            }.items()
+            if value
```

Comment:
This will probably keep default wdir value, which is not really desirable.

Reply:
Not really. `resolve_wdir` will return `None` if it's `os.curdir`.